Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla43527.cs

```diff
@@ -39,5 +39,24 @@ public TestPage()
 				};
 			}
 		}
+
+#if UITEST && __WINDOWS__
+		[Test]
+		public void TestB43527UpdateTitle()
+		{
+			try
+			{
+
+				RunningApp.WaitForElement(q => q.Marked("Change Title"));
+				RunningApp.WaitForElement(q => q.Marked("Test Page"));
+				RunningApp.Tap(q => q.Marked("Change Title"));
+				RunningApp.WaitForElement(q => q.Marked("Test Page"), timeoutMessage: "title changed. Element not found");
```

Comment:
Instead of catching the exception, you can just use `WaitForNoElement`.

Reply:
changed by using WaitForNotElement